src/main/java/seedu/duke/data/notebook/Note.java

```diff
@@ -60,14 +62,18 @@ public void setContent(String content) {
         this.content = content;
     }
 
-    public String getPinned() {
-        return (isPinned ? "Y" : "N");
```

Comment:
I think @prachi2023 has this line to display the info for pin note command. Can have the function as getPinnedString()